### ISSUE TITLE
Prayer wheel gave me duplicate legendary cards

### ISSUE THREAD

**Issue by MartinRykfors** [dbjorge/jorbs-spire-mod#435]
Slay the spire: Beta
Mod version: 1.4.0

After finishing a normal fight with the Prayer Wheel (Normal enemies drop an additional card reward), both of the card choices contained Patron. I was allowed to take both copies of it.

Thanks for making this mod, it is really fun!

**Comment by wang429** [comment]
Fully Reproduced using seed `205K8XCB4COCT`, Asc 2, Floor 1

**Comment by wang429** [comment]
I guess I should have also mentioned that I used the following to trim the card pool size down
```
        cards = Arrays.asList(
                new ForbiddenGrimoire(),
                new Strike_Wanderer(),
                new Defend_Wanderer(),
                new EyeOfTheStorm(),
                new FreshAdventure(),
                new Patron(),
                new FaerieFire(),
                new ColorSpray(),
                new FlameWard(),
                new ThirstingSword(),
                new Entangle(),
                new MageArmor(),
                new RayOfFrost(),
                new Banish(),
                new CorrodingBarrier(),
                new ArcaneWeapon(),
                new DisguiseSelf(),
                new ArcaneForm(),
                new ChatteringStrike());
```

**Comment by wang429** [comment]
Addressed by #440
